scripts/setup_ios_complete.sh

```
@@ -0,0 +1,517 @@
+while [[ ! $APPSTORE_ISSUER_ID =~ ^[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$ ]]; do
+    print_error "Invalid Issuer ID format. Must be UUID format (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx)"
+    read -p "Enter Issuer ID: " APPSTORE_ISSUER_ID
+done
+print_success "Issuer ID: $APPSTORE_ISSUER_ID"
+
+# API Key file
+print_step "4.3" "App Store Connect .p8 Key File"
+print_info "Looking for .p8 key file in current directory..."
+
+# Find .p8 files
+P8_FILES=($(find . -maxdepth 2 -name "AuthKey_*.p8" -o -name "*.p8" 2>/dev/null))
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)